src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = toFormat[3];
+  const dataArray = date.split(fromFormat[3]);
+  let toDate = '';
+
+  if (fromFormat.includes('YYYY') && toFormat.includes('YY')) {
+    const index = fromFormat.indexOf('YYYY');
+
+    fromFormat[index] = 'YY';
+    dataArray[index] = dataArray[index].slice(2);
+  }
+
+  if (fromFormat.includes('YY') && toFormat.includes('YYYY')) {
+    const index = fromFormat.indexOf('YY');
+
+    fromFormat[index] = 'YYYY';
+
+    if (dataArray[index] < 22) {
+      dataArray[index] = '20' + dataArray[index];
+    } else {
+      dataArray[index] = '19' + dataArray[index];
+    }
+  }
```

Comment:
```suggestion
   dataArray[index] = ( <ternary logic here> ) + dataArray[index];
```
Try to rewrite this in one simple line using ternary operator.

Reply:
corrected